### ISSUE TITLE
mozImageSmoothingEnabled shouldn't be used anymore

### ISSUE THREAD

**Issue by Selbi182** [1904labs/dom-to-image-more#119]
Hey, really nice tool and thank you so much for this update to the original library!

Just one minor thing, as a Firefox developer, it's kind of annoying that my console gets spammed with the following error:

![grafik](https://user-images.githubusercontent.com/8850085/222533124-4d7ddf16-9400-4364-b636-179ed83470d9.png)
Roughly translated: "mozImageSmoothingEnabled should no longer be used. Please use the imageSmoothingEnabled property without a prefix instead."

After manually downloading the source and commenting out that specific line, everything still worked fine. I'm assuming Firefox has included image smoothing to no longer require the Moz version.

 ```
function draw(domNode, options) {
        options = options || {};
        return toSvg(domNode, options)
            .then(util.makeImage)
            .then(function (image) {
                const scale = typeof options.scale !== 'number' ? 1 : options.scale;
                const canvas = newCanvas(domNode, scale);
                const ctx = canvas.getContext('2d');
                // ctx.mozImageSmoothingEnabled = false; // <- no longer necessary
                ctx.msImageSmoothingEnabled = false;
                ctx.imageSmoothingEnabled = false;
                if (image) {
                    ctx.scale(scale, scale);
                    ctx.drawImage(image, 0, 0);
                }
                return canvas;
            });
```

It's really a minor thing, but I'd appreciate if you could remove that line. :)

**Comment by IDisposable** [comment]
Nasty Firefox... we generally put all the browser-specific tweaks in and never take them out due to not wanting to force current browsers. I guess we should consider feature-testing for some of those.

**Comment by IDisposable** [comment]
Consider possible technique from #25

**Comment by Selbi182** [comment]
Hi, sorry for the late reply, wild week.

So, how exactly am I to use the technique you provided? Should I switch to that library? I kind of wish to stick with one I can simply [embed from cdnjs](https://cdnjs.com/libraries/dom-to-image-more) (I don't use npm).

**Comment by IDisposable** [comment]
That was a note-to-self about how to possibly fix this... still not sure we want to go down the browser-sniffing rabbit-hole.

**Comment by Selbi182** [comment]
I mean, for this particular case, isn't it simpler to just take a look at how long imageSmoothingEnabled has been part of Firefox?

According to [this page](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/imageSmoothingEnabled), it's been there since 2017. If anyone still uses a browser that hasn't been updated in six years (ignoring IE of course), I don't think image smoothing should be their top concern.

**Comment by IDisposable** [comment]
Fair point... want to document the other browser's history so I know what's safe? I can't reasonably test against anything more than Chrome in the regression tests.

**Comment by IDisposable** [comment]
I've removed this in 3.1.1

**Comment by Selbi182** [comment]
I can't see the new version on cdnjs yet. When do you think it will appear?

**Comment by IDisposable** [comment]
both v3.1.1 and v2.16.0 have the fix and are on https://www.npmjs.com/package/dom-to-image-more?activeTab=versions
